packages/unmock-core/src/__tests__/generator.test.ts

```diff
@@ -1,7 +1,8 @@
 import fs from "fs";
 import yaml from "js-yaml";
 import { mockGeneratorFactory } from "../generator";
-import { ISerializedRequest, OASSchema, RequestToSpec } from "../interfaces";
+import { ISerializedRequest, RequestToSpec } from "../interfaces";
```

Comment:
When we'll get to a more generator-specific PR, these would probably change (i.e. `RequestToSpec` feels out of place at the moment)